src/test/kotlin/three/consulting/epoc/controller/TimesheetEntryControllerIntegrationTest.kt

```diff
@@ -20,13 +20,13 @@ class TimesheetEntryControllerIntegrationTest : ControllerIntegrationTest() {
     @Test
     fun `add timesheetEntry returns 200`() {
         val httpEntity = jsonPostEntity("timesheetEntry/validCreation.json")
-        val response = restTemplate.postForEntity("/timesheet-entry", httpEntity, ObjectNode::class.java)
+        val response = restTemplate.postForEntity("/timesheet-entry", httpEntity, ArrayNode::class.java)
         assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
     }
     @Test
     fun `add timesheetEntries returns 200`() {
         val httpEntity = jsonPostEntity("timesheetEntry/validCreateMany.json")
-        val response = restTemplate.postForEntity("/timesheet-entries", httpEntity, ArrayNode::class.java)
+        val response = restTemplate.postForEntity("/timesheet-entry", httpEntity, ArrayNode::class.java)
```

Comment:
The two tests seem to be very much identical so you can remove one of them if you want